volatility3/framework/plugins/windows/vadinfo.py

```diff
@@ -274,6 +300,24 @@ def filter_function(x: interfaces.objects.ObjectInterface) -> bool:
 
     def run(self) -> renderers.TreeGrid:
         filter_func = pslist.PsList.create_pid_filter(self.config.get("pid", None))
+        kernel = self.context.modules[self.config["kernel"]]
+
+        if self.config["offset"]:
+            procs = psscan.PsScan.scan_processes(
+                self.context,
+                self.config["kernel"],
```

Comment:
```suggestion
                kernel.layer_name,
```
Why do you go back to the config option here, but a couple of lines later use `kernel.layer_name`.  They *should* both be the same thing, but if they're not for some reason, this will cause problems.  Make sure they're both `kernel.layer_name`.